### DISCUSSION TITLE
Change device information (GPU vendor/renderer)?

### DISCUSSION THREAD

**Discussion by FlowFlaux** [seleniumbase/SeleniumBase#3109] [Q&A] [answered]
Is there a way to alter the device information being provided by the Chrome instance being controlled by SB? I have a suspicion that a site I am trying to scrape might be using the info about my machine's GPU to detect bots or at the very least assign higher risk scores if there isn't one.

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/discussions/2248#discussioncomment-7501491.

Also see https://github.com/seleniumbase/SeleniumBase/issues/2404.
